src/frontend/Sider/Behandling/Inngangsvilkår/Aktivitet/validering.ts

```diff
@@ -0,0 +1,29 @@
+import { NyAktivitet } from './LeggTilAktivitet';
+import { FormErrors } from '../../../../hooks/felles/useFormState';
+import { erDatoEtterEllerLik } from '../../../../utils/dato';
+
+export const validerForm = ({ fom, tom, type }: NyAktivitet): FormErrors<NyAktivitet> => {
+    const feil: FormErrors<NyAktivitet> = {
+        fom: undefined,
+        tom: undefined,
+        type: undefined,
+    };
```

Comment:
Vi lagde en type for periode som man kunne extende. Kan denne brukes så det blir `NyAktivitet extends Periode`?
Og så kunne vi vurdert å lage og så snakket vi om å lage en metode `erFomFørTom` eller bare `validerPeriode` i `dato.ts` som tar inn to perioder og kan brukes alle steder i stedet. Siden vi har samme sjekk mange steder nå

Reply:
Bra innspill 👍 
Måtte endre Periode til type i stedet for interface for at dette skulle virke pga quirk med interfaces, extends og "structural type compability"
https://nav-it.slack.com/archives/C6HJFRRMY/p1703153698542089